src/content/docs/dev/send-tokens/sui-its.mdx

```diff
@@ -0,0 +1,502 @@
+
+To integrate a coin to ITS the `register_coin()` function must be called. This function takes in the following parameters:
+
+1. `its`: The ITS module that will be updated once the new coin is registered
+1. `coin_info`: The [coin info](#coininfo) represents the of the coin
+1. `coin_management`: The type of [management](#coinmanagement) the coin will have with ITS
+
+Before you can register your coin however, you first must create a [coin-manager]("coin-manager") and a [coin-info]("coin-info") for your coin.
+
+```move
+public fun register_coin<T>(
+    self: &mut ITS,
+    coin_info: CoinInfo<T>,
+    coin_management: CoinManagement<T>,
+): TokenId {
+    let value = self.value_mut!(b"register_coin");
+
+    value.register_coin(coin_info, coin_management)
+}
+```
+
+Once triggered, ITS will generate a unique [token id](#tokenid) from the coin's metadata and management details. Then it will [add](https://docs.sui.io/references/framework/sui/bag#sui_bag_add) the tokenId to the `registered_coins` bag. The full implementation of the `register_coin()` function can be found [here](https://github.com/axelarnetwork/axelar-cgp-sui/blob/v1.1.3/move/interchain_token_service/sources/versioned/interchain_token_service_v0.move#L166)
+
+See [here](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/sui/its-example.js#L195) for an example of how to register a new coin.
+
+### Deploy Remote Interchain Token
+
+If you are starting from scratch and want to deploy a fresh new token with cross-chain functionality built into it, you can trigger the `deploy_remote_interchain_token()` function. This function will deploy a new *Interchain Token* on a different blockchain via a cross-chain [GMP call](https://docs.axelar.dev/dev/general-message-passing/overview/#general-message-passing). 
+
+The function takes three parameters:
+
+1. `its`: The ITS module that will be updated once the new token is registered
+1. `token`: The [token_id](#tokenid) representing the token to be deployed on the destination chain
+1. `destination_chain`: The name of the destination chain to deploy the token on
+
+```move
+public fun deploy_remote_interchain_token<T>(
+    self: &ITS,
+    token_id: TokenId,
+    destination_chain: String,
+): MessageTicket {
+    let value = self.value!(b"deploy_remote_interchain_token");
+
+    value.deploy_remote_interchain_token<T>(token_id, destination_chain)
+}
+```
+
+Since this function is making a cross-chain call, it will return a [MessageTicket](TODO) for the cross-chain transaction. The full implementation of the `deploy_remote_interchain_token()` can be found [here](https://github.com/axelarnetwork/axelar-cgp-sui/blob/v1.1.3/move/interchain_token_service/sources/versioned/interchain_token_service_v0.move#L178).
+
+See here for an [example](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/sui/its-example.js#L107) of how to run a cross-chain deployment for a new token.
+
+### Interchain Transfer
```

Comment:
Mention that apps that wish to send tokens should prepare interchain transfer and return the `InterchainTransferTicket` and have the frontend call `send_interchain_transfer` separately in the same Transaction Block. This makes ITS more easily upgradable without breaking apps upon upgrade.